Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM amazoncorretto:21.0.1-alpine
```

Comment:
Could this use the CH amazonlinux:2023 runtime/build jobs for corretto jdk 21?

416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-build-21
416670754337.dkr.ecr.eu-west-2.amazonaws.com/ci-corretto-runtime-21

Reply:
Done